cmake/PlasmaTargets.cmake

```diff
@@ -0,0 +1,84 @@
+include(BuildOptimizations)
+include(QtDeployment)
+#TODO (#854): include(Sanitizers)
+
+function(plasma_executable TARGET)
+    cmake_parse_arguments(_pex
+        "WIN32;CLIENT;TOOL;QT_GUI;EXCLUDE_FROM_ALL"
+        ""
+        "SOURCES"
+        ${ARGN}
+    )

```

Comment:
Since plasma_test forwards to this, we may want to use the [PARSE_ARGV ](https://cmake.org/cmake/help/v3.14/command/cmake_parse_arguments.html?highlight=parse_argv) form of this function to be safe. The documentation calls out `;`, but I have read about issues with `ARGN` stomping on other websites.